<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBMS - Dense Index</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutControls">
      <item>
       <widget class="QGroupBox" name="groupBoxInput">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelKey">
           <property name="text">
            <string>Key:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinKey">
           <property name="maximum">
            <number>2147483647</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelData">
           <property name="text">
            <string>Data:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="editData"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxOps">
        <property name="title">
         <string>Operations</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutOps">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEdit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxTools">
        <property name="title">
         <string>Tools</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutTools">
         <item>
          <widget class="QPushButton" name="btnGenerate">
           <property name="text">
            <string>Generate 10k Random</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInfo">
           <property name="text">
            <string>Show Info / Analysis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>Clear Database</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutDisplay">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabIndex">
         <attribute name="title">
          <string>Index Table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayoutIndex">
          <item>
           <widget class="QTableWidget" name="tableIndex">
            <column>
             <property name="text">
              <string>Key</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Offset</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabData">
         <attribute name="title">
          <string>Data Table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayoutData">
          <item>
           <widget class="QTableWidget" name="tableData">
            <column>
             <property name="text">
              <string>Key</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Data</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxLog">
        <property name="title">
         <string>Log / Output</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutLog">
         <item>
          <widget class="QTextEdit" name="textLog">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
